docs/reference/phi.md

```diff
@@ -6,15 +6,20 @@ The reader supports [Phi MultiPak](https://www.phi.com/surface-analysis-equipmen
 
 The reader for the Phi data can be found [here](https://github.com/FAIRmat-NFDI/pynxtools-xps/tree/main/pynxtools_xps/phi).
 
-Example data for this file format is available [here](https://github.com/FAIRmat-NFDI/pynxtools-xps/tree/main/examples/phi).
+## .spe data (single spectrum):
 
-The example conversion can be run with the following commands:
+Example data for this file format is available [here](https://github.com/FAIRmat-NFDI/pynxtools-xps/tree/main/examples/phi).
 
-### For the .spe data (single spectrum):
+The example conversion can be run with the following command:
 ```console_
 user@box:~$ dataconverter SnO2_10nm.spe eln_data_phi.yaml --reader xps --nxdl NXmpes --output SnO2_10nm.spe.nxs
 ```
-### For the .pro data (profiling):
+
+### .pro data (profiling):
+Example data for this file format is available [here](https://github.com/FAIRmat-NFDI/pynxtools-xps/tree/main/examples/phi).
```

Comment:
Is it normal that you have the same link as for .spe files?

Reply:
Yeah, the data is all in the same folder as for both, I am using the same reader and the same ELN file.